src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -91,6 +99,22 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
     nbCount++;
   }
 
+  public get leftReorderDroppable() {
+    return ColumnHeaderSides.LEFT;
+  }
+
+  public get rightReorderDroppable() {
+    return ColumnHeaderSides.RIGHT;
+  }
+
+  public get columnDropData() {
+    return this.columnOrderModel;
+  }
+
+  public get columnOrderDropKey() {
```

Comment:
Naming nitpick: you're using group/groupId everywhere, "dropKey" is inconsistent.

Reply:
Changed to `columnGroupId` everywhere.